package.json

```diff
@@ -19,7 +19,10 @@
     "lint": "gulp lint",
     "build": "gulp build",
     "dist": "gulp dist",
-    "heroku-postbuild": "gulp clean && gulp build && gulp dist --fortesting"
+    "heroku-postbuild": "gulp clean && gulp build --fortesting && gulp dist --fortesting && npm run try-prepend",
+    "try-prepend": "if [ -n \"$AMP_TESTING_HOST\" ]; then npm run prepend-amp; npm run prepend-v0; fi",
+    "prepend-amp": "gulp prepend-global --prod node_modules/AMP_CONFIG.json --local --target dist/amp.js && gulp prepend-global --prod node_modules/AMP_CONFIG.json --local --target dist.3p/current/integration.js",
+    "prepend-v0": "gulp prepend-global --prod node_modules/AMP_CONFIG.json --local --target dist/v0.js && gulp prepend-global --prod node_modules/AMP_CONFIG.json --local --target dist.3p/current-min/f.js"
```

Comment:
@cramforce made the changes. 

I did realize though that the AMP_CONFIG url overriding doesn't work in production right now cause we never prepend the global configs to `f.js`. should we be prepending it?

cc @dknecht